crates/sui-bridge/src/sui_bridge_watchdog/sui_bridge_status.rs

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+//! The SuiBridgeStatus observable monitors whether the Sui Bridge is paused.
+
+use crate::sui_bridge_watchdog::Observable;
+use crate::sui_client::SuiBridgeClient;
+use async_trait::async_trait;
+use prometheus::IntGauge;
+use std::sync::Arc;
+
+use tokio::time::Duration;
+use tracing::{error, info};
+
+pub struct SuiBridgeStatus {
+    sui_client: Arc<SuiBridgeClient>,
+    metric: IntGauge,
+}
+
+impl SuiBridgeStatus {
+    pub fn new(sui_client: Arc<SuiBridgeClient>, metric: IntGauge) -> Self {
+        Self { sui_client, metric }
+    }
+}
+
+#[async_trait]
+impl Observable for SuiBridgeStatus {
+    fn name(&self) -> &str {
+        "SuiBridgeStatus"
+    }
+
+    async fn observe_and_report(&self) {
+        let status = self.sui_client.is_bridge_paused().await;
+        match status {
+            Ok(status) => {
+                self.metric.set(status as i64);
+                info!("Sui Bridge Status: {:?}", status);
+            }
+            Err(e) => {
+                error!("Error getting sui bridge status: {:?}", e);
+            }
+        }
+    }
+
+    fn interval(&self) -> Duration {
+        Duration::from_secs(2)
```

Comment:
should this match eth_bridge_status duration?

Reply:
see https://github.com/MystenLabs/sui/pull/19878#discussion_r1806800346